mitodl/odl-video-service
# Email notifications

#### What are the relevant tickets?
closes #64 

#### What's this PR do?
Add support to send email notifications every time the video get to a final state

#### How should this be manually tested?
Add a valid email to your user, upload a video (or something to make something fail), check the mailbox.


NOTE: `mail.api`, `mail.utils` and relative test files have been copied from other projects so can be skipped during the review
